iotlabcli/parser/profile.py

```diff
@@ -280,9 +280,11 @@ def _custom_profile(opts):
 def _add_profile(api, name, profile, json_out=False):
     """ Add user profile. if json, dump json dict to stdout """
     if json_out:
-        return profile
+        ret = profile
     else:
-        return api.add_profile(name, profile)
```

Comment:
Here, the "else" clause can be removed totally and just do `return api.add_profile`.
It would better emphasize the fact that `json_out` is a special case handling.